pwndbg/commands/next.py

```diff
@@ -36,6 +36,13 @@ def nextcall(*args):
 
 @pwndbg.commands.Command
 @pwndbg.commands.OnlyWhenRunning
+def nextsymbol(*args):
```

Comment:
Ok, gladly do that. How should I differentiate if an address or regex parameter was given?